### ISSUE TITLE
resourceRender on resource agenda, wrong data

### ISSUE THREAD

**Issue by Efiware** [fullcalendar/fullcalendar-scheduler#91]
When using the new vertical resource (agenda) view and implementing the resourceRender method, I get a funny error: All the column titles are identical!
Example code:
```
<!DOCTYPE html>
<html>
<head>
    <meta charset='utf-8' />
    <link href='//fullcalendar.io/js/fullcalendar-scheduler-1.1.0/lib/fullcalendar.min.css' rel='stylesheet' />
    <link href='//fullcalendar.io/js/fullcalendar-scheduler-1.1.0/lib/fullcalendar.print.css' rel='stylesheet' media='print' />
    <link href='//fullcalendar.io/js/fullcalendar-scheduler-1.1.0/scheduler.min.css' rel='stylesheet' />
    <script src='//fullcalendar.io/js/fullcalendar-scheduler-1.1.0/lib/moment.min.js'></script>
    <script src='//fullcalendar.io/js/fullcalendar-scheduler-1.1.0/lib/jquery.min.js'></script>
    <script src='//fullcalendar.io/js/fullcalendar-scheduler-1.1.0/lib/fullcalendar.min.js'></script>
    <script src='//fullcalendar.io/js/fullcalendar-scheduler-1.1.0/scheduler.min.js'></script>
    <script>

        $(function() { // document ready

            $('#calendar').fullCalendar({
                defaultView: 'agendaDay',
                defaultDate: '2015-12-07',
                editable: true,
                selectable: true,
                eventLimit: true,
                header: {
                    left: 'prev,next today',
                    center: 'title',
                    right: 'agendaDay,agendaSevenDay'
                },
                views: {
                    agendaSevenDay: {
                        type: 'agenda',
                        duration: { days: 7 },
                        groupByResource: true
                    }
                },

                //// uncomment this line to hide the all-day slot
                //allDaySlot: false,


                events: [
                    { id: '1', resourceId: 'a', start: '2015-12-06', end: '2015-12-08', title: 'event 1' },
                    { id: '2', resourceId: 'a', start: '2015-12-07T09:00:00', end: '2015-12-07T14:00:00', title: 'event 2' },
                    { id: '3', resourceId: 'b', start: '2015-12-07T12:00:00', end: '2015-12-08T06:00:00', title: 'event 3' },
                    { id: '4', resourceId: 'c', start: '2015-12-07T07:30:00', end: '2015-12-07T09:30:00', title: 'event 4' },
                    { id: '5', resourceId: 'd', start: '2015-12-07T10:00:00', end: '2015-12-07T15:00:00', title: 'event 5' }
                ],

                resources: function (callBack) {
                    callBack([
                        { id: 'a', title: 'Room A', altTitle: 'Super Room A' },
                        { id: 'b', title: 'Room B', altTitle: 'Super Room B' , eventColor: 'green' },
                        { id: 'c', title: 'Room C', altTitle: 'Super Room C' , eventColor: 'orange' },
                        { id: 'd', title: 'Room D', altTitle: 'Super Room D' , eventColor: 'red' }
                    ]);
                },
                resourceRender: function (dataTds, eventTd) {
                    console.log('resourcerender');
                    console.log(eventTd);
                    console.log(dataTds);
                    var textElement = eventTd.empty();
                    textElement.append('<b>' + dataTds.altTitle + '</b>');
                },
            });

        });

    </script>
    <style>

        body {
            margin: 0;
            padding: 0;
            font-family: "Lucida Grande",Helvetica,Arial,Verdana,sans-serif;
            font-size: 14px;
        }

        #calendar {
            max-width: 900px;
            margin: 50px auto;
        }

    </style>
</head>
<body>

<div id='calendar'></div>

</body>
</html>
```

Result:

![Image of Error](https://lh3.googleusercontent.com/X34XsdYAFbwOEu-5YwF6xtMDqlWlVA5MNF1fiLVKCW-MLZHMNmr7Z-FoEpQ3NAEfU73TJg=w1896-h835)

**Comment by arshaw** [comment]
hey @Efiware would you mind getting this into a JSBin?
http://fullcalendar.io/wiki/Reporting-Bugs/
"Scheduler Add-on" > "Simple Template"

**Comment by Efiware** [comment]
Sure, JSbin here:

http://jsbin.com/gopaginoli/1/edit?js,output

**Comment by arshaw** [comment]
Are you sure? Here's what I'm seeing in my console:

![screen shot 2016-01-11 at 10 54 28 pm](https://cloud.githubusercontent.com/assets/56520/12256903/579397ae-b8b6-11e5-8e45-e16ed58ba29c.png)

As you can see, each `title` property is different

**Comment by Efiware** [comment]
Did you press the week button? After I press this button, my console looks like this:

![image](https://cloud.githubusercontent.com/assets/10004048/12258255/91e10412-b90d-11e5-8af2-f2f8a8d41e6f.png)

**Comment by arshaw** [comment]
i did not, im sorry for that. i see the problem now. that is bizarre. I will get this fixed

**Comment by arshaw** [comment]
updated this jsbin with latest. fixed: http://jsbin.com/vameyucefi/1/edit?js,output

**Comment by arshaw** [comment]
just released in [v1.2.1](https://github.com/fullcalendar/fullcalendar-scheduler/releases/tag/v1.2.1)
